src/cr/cube/cube_slice.py

```diff
@@ -453,6 +482,27 @@ def _scalar_type_std_res(self, counts, total, colsum, rowsum):
         )
         return residuals / np.sqrt(variance)
 
+    def _categorical_pairwise_chisq(self, counts, axis=0):
+        """Return ndarray containing pairwise comparisons along axis
+
+        Returns a square, symmetric matrix of test statistics for the null
+        hypothesis that each vector along *axis* is equal to each other.
+        """
+        if axis == 0:
+            margin = self.margin(0)
+            offmargin = self.margin(1)
+            proportions = self.proportions(axis=0)
+            wts = offmargin / self.margin()
+            elements = counts.shape[1]
+            chisq = np.zeros([elements, elements])
+            for i in xrange(1, elements):
+                for j in xrange(0, elements - 1):
+                    chisq[i, j] = chisq[j, i] = np.sum(
+                        np.square(proportions[:, i] - proportions[:, j]) / wts
+                    ) / (1 / margin[i] + 1 / margin[j])
+
+            return chisq
+
```

Comment:
There should be an explicit return path, or a raise, which should in turn be covered by a unit test :)

Reply:
fine with the same `NotImplementedError`, or we can do the index-twiddling that the R impl does and test with the transpose.

Reply:
Is that the source of my minuscule coverage drop? oops